locale/en.po

```diff
@@ -4203,7 +4203,11 @@ msgstr "This panel shows the numbers that the auto-evo prediction works from. Th
 msgid "COMPOUND_BALANCE_TITLE"
 msgstr "Compound Balances"
 
-#: ../src/microbe_stage/editor/EditorComponentBottomLeftButtons.cs:259
+#: ../src/microbe_stage/editor/EditorComponentBottomLeftButtons.cs:263
+msgid "SPECIES_NAME_TOO_LONG_POPUP"
+msgstr "Species name must fit in the box!"
```

Comment:
I think this text should be just that the species name is too long. I think at most it should be like "Species name is too long (it must fit in the text box)".